test/e2e/helpers/visu/playwright-utils.ts

```diff
@@ -48,36 +47,3 @@ export async function chromiumMouseZoom(x: number, y: number, deltaX: number): P
   // detach is it not reused outside this function
   await client.detach();
 }
-
-// TODO To remove when https://github.com/microsoft/playwright/issues/1094 is fixed
-export async function webkitMousePanning(panningOptions: PanningOptions): Promise<void> {
```

Comment:
I prefer to not do to much change in this PR and remove this in another.

Reply:
I was unable to make this works with the new playwright version.
You can check the commit history.

The WebKit mouse panning hack required change (incompatible type between dom element and ElementHandle) that we would have removed anyway (see #

Reply:
Ok, I didn't see the PR failed in first place.
So, I'm ok to merge like that 🙂